### PR TITLE
[Orca] Update Documents for Tf2estimator on Pyspark Backend

### PR DESCRIPTION
**Pull request by sgwhat**
Update distributed training docs for TF2Estimator on pyspark backend updating.

### CODE REVIEW

File: docs/readthedocs/source/doc/Orca/Overview/distributed-training-inference.md [hunk 1]
```diff
@@ -97,6 +100,57 @@ The `data` argument in `fit` method can be a spark DataFrame, an *XShards* or a
...
+                epochs=max_epoch,
+                steps_per_epoch=total_size // batch_size)
+predictions = est.predict(data=df,
+                          feature_cols=['image']).collect()
+```
+
+The `data` argument in `fit` method can be a spark DataFrame, an *XShards* or a *Data Creator Function* (that returns a `tf.data.Dataset`). The `data` argument in `predict` method can be a spark DataFrame or an *XShards*. See the *data-parallel processing pipeline* [page](./data-parallel-processing.md) for more details.
+
+View the related [Python API doc]() for more details.
+
```
Comment (jenniew) [inline docs/readthedocs/source/doc/Orca/Overview/distributed-training-inference.md:153]: where is [Python API doc]? Can you add tf2 pyspark_estimator and ray_estimator API to Orca API docs, and put correct link here.

File: docs/readthedocs/source/doc/Orca/Overview/distributed-training-inference.md [hunk 2]
```diff
@@ -97,6 +100,57 @@ The `data` argument in `fit` method can be a spark DataFrame, an *XShards* or a
...
 ***For more details, view the distributed TensorFlow training/inference [page]().***
```
Comment (jenniew) [inline docs/readthedocs/source/doc/Orca/Overview/distributed-training-inference.md:154]: If "distributed TensorFlow training/inference" doc is not available now, please put todo link here just like what pytorch estimator did.

File: docs/readthedocs/source/doc/Orca/Overview/distributed-training-inference.md [hunk 3]
```diff
@@ -97,6 +100,57 @@ The `data` argument in `fit` method can be a spark DataFrame, an *XShards* or a
...
+    return args
+
+est = Estimator.from_keras(model_creator=model_creator,
+                           config={"lr": 1e-2},
+                           workers_per_node=2,
+                           backend="spark",
+                           model_dir=model_dir)
+```
+
+The `model_creator` argument should be a function that takes a `config_args` dictionary and returns a compiled Keras model.
```
Comment (jenniew) [inline docs/readthedocs/source/doc/Orca/Overview/distributed-training-inference.md:132]: 'config' dictionary

File: docs/readthedocs/source/doc/Orca/Overview/distributed-training-inference.md (hunk 1 above)
Comment (sgwhat) [reply docs/readthedocs/source/doc/Orca/Overview/distributed-training-inference.md:153]: Seems that's an error existed before, will fix soon.

**Comment by sgwhat** [comment]
Does any other place need to be updated? @jenniew

**Review by jenniew — APPROVED** [review]
LGTM

**Pull request merged**